composer.json

```diff
@@ -36,7 +36,7 @@
   ],
   "minimum-stability": "dev",
   "require": {
-    "nosto/php-sdk": "3.10.0"
+    "nosto/php-sdk": "dev-hotfix/3.10.1"
```

Comment:
Why the dev branch?

Reply:
For the CI not to fail, it will be changed back when the SDK PR will be merged to master